Kitodo/src/main/java/org/kitodo/workflow/model/Updater.java

```diff
@@ -40,8 +40,6 @@ public void updateProcessesAssignedToTemplate() throws DataException {
         List<Process> processes = this.template.getProcesses();
 
         for (Process process : processes) {
-            process.setDocket(this.template.getDocket());
-            process.setRuleset(this.template.getRuleset());
```

Comment:
You can remove following empty line too.

Reply:
Removed